# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/dietlibc/__i686-defined-hotfix.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

taken from T2 trunk
2005-01-17 daja77

While optimizing for pentiumII, __i686 gets defined, resulting
in a compile error "junk at end of line"...
	-- jsaw

--- dietlibc-0.27/i386/PIC.h.orig	2004-12-12 00:41:23.957319464 +0100
+++ dietlibc-0.27/i386/PIC.h	2004-12-12 01:04:36.086683632 +0100
@@ -2,6 +2,8 @@
 #define __I386_PIC_H__
 #ifdef __DYN_LIB
 
+#undef __i686
+
 #if 1
 /* don't trash the athlon return stack */
 #if (__GNUC__ >= 3)	// FIXME: how do I check for the binutils version ?
